boards/arm/mm_swiftio/burner/pyocd_user.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020, MADMACHINE LIMITED
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+def will_connect():
+    # Look up the external flash memory region.
+    extFlash = target.memory_map.get_region_by_name("flexspi")
```

Comment:
Does adding 'global target' before this line fix your pylint issue?

Reply:
Thanks so much. I fixed it. I'm a beginner in python and don't know how to deal with this problem, now you provide the answer to me。 I'll close https://lists.zephyrproject.org/g/devel/topic/add_pyocd_user_scripy_cause/71021588?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,71021588

Reply:
It looks like you can do something like this:

```
        # 'target' is global
        # pylint: disable=undefined-variable
```

See here for an example: https://github.com/zephyrproject-rtos/ci-tools/pull/128/files

Reply:
Thanks so mush, It be fixed